spec/presenters/reports/boxi/registration_presenter_spec.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+module Reports
+  module Boxi
+    RSpec.describe ::Reports::Boxi::RegistrationPresenter do
+      let(:registration) { double(:registration) }
+
+      subject { described_class.new(registration, nil) }
+
+      describe "#finance_details_balance" do
+        it "returns a formatted date" do
```

Comment:
I'm thinking it does this 😁 !

```suggestion
        it "returns the balance formatted as pounds and pence" do
```

Reply:
Not really, I went back and fixed the usual one, but didn't spot this!